x/ccv/provider/types/codec.go

```diff
@@ -11,6 +12,7 @@ import (
 // RegisterLegacyAminoCodec registers the necessary x/ibc transfer interfaces and concrete types
 // on the provided LegacyAmino codec. These types are used for Amino JSON serialization.
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
+	cdc.RegisterConcrete(&MsgAssignConsumerKey{}, "ccv/MsgAssignConsumerKey", nil)
```

Comment:
```suggestion
	cdc.RegisterConcrete(&MsgAssignConsumerKey{}, "/interchain_security.ccv.provider.v1.MsgAssignConsumerKey", nil)
```
should we use full path here?